lib/containers/common.pm

```diff
@@ -334,6 +334,8 @@ sub switch_cgroup_version {
 
     my $setting = ($version == 1) ? 0 : 1;
 
+    return if (script_run("grep -q systemd\.unified_cgroup_hierarchy=$setting /proc/cmdline") == 0);
```

Comment:
```suggestion
    return if (script_output("/proc/cmdline") =~ "systemd\.unified_cgroup_hierarchy=$setting");
```
Better to use `script_output` because we can see the output in the logs if needed.